diag_manager/fms_diag_file_object.F90

```diff
@@ -123,6 +124,7 @@ module fms_diag_file_object_mod
 type, extends (fmsDiagFile_type) :: subRegionalFile_type
   integer, dimension(:), allocatable :: sub_axis_ids !< Array of axis ids in the file
   logical :: write_on_this_pe !< Flag indicating if the subregion is on the current PE
+  logical :: subaxis_defined !< Flag indicating if the subaxes have already been defined
```

Comment:
This is kinda nitpicky,  but I would prefer a variable like this named `is_subaxis_defined`.  I don't think it's absolutely necessary to change this though.